scripts/gatsby-source-iiif/indexSearch.js

```diff
@@ -140,15 +136,15 @@ const getSearchDataFromManifest = (manifest) => {
     search['formatTag'] = [manifest.workType]
   }
 
-  search['allMetadata'] = ''
+  search['allMetadata'] = []
   allMetadataKeys.forEach((key) => {
     if (manifest[key]) {
-      search['allMetadata'] += '::' + manifest[key]
+      search['allMetadata'].push(manifest[key])
     }
   })
-  search['allMetadata'] += '::' + loadSubItemTitles(manifest)
-  search['allMetadata'] += '::' + search.centuryTag.join('::')
-  search['allMetadata'] += '::' + search.themeTag.join('::')
```

Comment:
This magic string was used somewhere in the frontend too at one point? is it still there and need remediated?